### PR TITLE
Add a simple java Hello World application using HBase API

### PR DESCRIPTION
**Pull request by garye**
Resubmitting hello world app with profile activation in pom.xml.

### CODE REVIEW

File: java/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Run the application
+
+First, set your [Google Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
+and [install Maven](http://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) if necessary.
+
+
+    $ mvn package
+    $ mvn exec:java -Dexec.args="<projectId> <zone> <clusterId>"
+
+You will see output resembling the following, interspersed with informational logging
```
Comment (lesv) [inline java/hello-world/README.md]: You need to mention the profiles here.  -Pmac -Pwindows, etc.

**Comment by lesv** [comment]
Just add a mention of the profiles to the README, and I'm happy.

You might wish to verify, if Disable is required for Bigtable, if not, you may wish to comment that it is optional for bigtable, but required for HBase.

File: java/hello-world/README.md (hunk 1 above)
Comment (garye) [inline java/hello-world/README.md]: @sduskis This is also relevant to your comment on the outdated PR, but I added the <activation/> tag for each profile that automatically chooses the correct profile based on on the os family. Seems to work well on my mac:

garyelliott-macbookpro:hello-world garyelliott$ mvn help:active-profiles
[INFO] 
Active Profiles for Project 'com.example.bigtable.helloworld:cloud-bigtable-hello-world:jar:1.0-SNAPSHOT': 

The following profiles are active:

 - mac (source: com.example.bigtable.helloworld:cloud-bigtable-hello-world:1.0-SNAPSHOT)

**Comment by lesv** [comment]
LGTM - Though I'd like it to match the other apps in the future and use -Dbigtable.clusterID= -Dbigtable.projectID= and -Dbigtable.zone=

**Pull request merged**